lucene/suggest/src/java/org/apache/lucene/search/suggest/document/TopSuggestDocsCollector.java

```diff
@@ -201,4 +204,19 @@ public void collect(int doc) throws IOException {
   public ScoreMode scoreMode() {
     return ScoreMode.COMPLETE;
   }
+
+  /**
+   * returns true if the collector clearly exhausted all possibilities to collect results
+   */
+  public boolean isComplete() {
```

Comment:
Can we make this package protected and add a boolean in `TopSuggestDocs` that indicates if the search is complete or not ?